Update distribution.md (#547)
codemagic-ci-cd/codemagic-docs: content/yaml-publishing/distribution.md

You can override the publishing track specified in the configuration file using the environment variable `GOOGLE_PLAY_TRACK`. This is useful if you're starting your builds via [Codemagic API](../rest-api/overview/) and want to build different configurations without editing the configuration file.
{{</notebox>}}

{{<notebox>}}

> Multiple Google Play Console accounts accounts are allowed to be connected for publishing other android apps with different workflows.

Reviewer: ```suggestion
To use different Google Play Console accounts for publishing your Android apps, set up separate workflows. 
```

```

{{<notebox>}}

> Multiple Apple Developer Portal accounts can be connected for publishing other iOS apps with different workflows

Reviewer: To use different Apple Developer Portal accounts for publishing your iOS apps, set up separate workflows.

{{</notebox>}}

> #### Post-processing of App Store Connect distribution or Magic Actions

Reviewer: ```suggestion
#### Post-processing of App Store Connect distribution (Magic Actions)
```

Some App Store Connect actions, like `submit_to_testflight`, `beta_groups` and uploading release notes take place asynchronously in the post-processing step after the app artifact has been successfully published to App Store Connect and the main workflow has completed running in Codemagic. This avoids using the macOS build machine while we are waiting for Apple to complete processing the build and it becomes available for further actions. 

> Post-processing of App Store Distribution jobs or Magic Actions in short has a two-step timeout. If the uploaded build cannot be found in App Store Connect in 15 minutes, the step times out. This may happen if there are issues with the uploaded artifact, in which case the build does not become available in App Store Connect at all and you'll receive an email from App Store Connect. The overall timeout for post-processing is 120 minutes. If the uploaded build has not exited the processing status by then, post-processing in cancelled. You will be still able to manually submit the build to beta review, upload release notes and distribute the app to beta groups once the build becomes available in App Store Connect.

Reviewer 2: ```suggestion
Post-processing of App Store Distribution jobs, or Magic Actions in short, has a two-step timeout. If the uploaded build cannot be found in App Store Connect in 15 minutes, the step times out. This may happen if there are issues with the uploaded artifact, in which case the build does not become available in App Store Connect at all and you'll receive an email from App Store Connect. The overall timeout for post-processing is 120 minutes. If the uploaded build has not exited the processing status by then, post-processing is cancelled. You will still be able to manually submit the build to beta review, upload release notes and distribute the app to beta groups once the build becomes available in App Store Connect.
```
